tests/bitset.test.cpp

```diff
@@ -0,0 +1,202 @@
+#include <boost/ut.hpp>
+#include <cstdint>
+#include <libxbitset/bitset.hpp>
+
+namespace embed {
```

Comment:
```suggestion
namespace xstd {
```

`xstd` is the correct namespace here, not embed.

Reply:
This will allow you to remove the `xstd::` infront of the `bitset` objects.